0001_cars_test.cy.js

```diff
@@ -382,20 +385,16 @@ describe("Test Child:", () => {
       ).click();
       cy.get(
          cyInterfaceCARS.page.socialWorker.page.children.form.addChildren.option
-            .project[0]
+            .carsProject[0]
       )
          .should("be.visible")
          .click();
       cy.get(
          cyInterfaceCARS.page.socialWorker.page.children.form.addChildren.button
             .save
-      ).click();
-
-      cy.get(
```

Comment:
The reason we'd check that a button isn't visible is to make sure the popup has closed before trying to continue.
If the test passes consistently it's fine though.